src/content/docs/browser/new-relic-browser/configuration/consent-mode.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Consent mode
+tags:
+  - Browser
+  - Browser monitoring
+metaDescription: "Integrate privacy policy opt-ins to let users decide whether to consent to allow the browser agent to send data over the network."
+freshnessValidatedDate: never
+---
+
+## What does this feature do? 
+
+This allows for the integration of privacy policy opt-ins for your users on how web applications collect, transmit, and process data monitored by the browser agent. By enabling this feature, you can let users decide whether to consent to allow the browser agent to send data over the network.
+
+A new property is available in the browser agent’s init object called `useConsentMode`. This setting controls whether to use the consent mode in the agent. Available in the Browser Lite, Browser Pro or Pro + SPA agent, [version 1.303.0](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1.303.0) or greater.
+
+## How do I use it?
+
+To use this feature, you will need to visit your app settings page and turn the Consent Mode toggle to On. This setting is off by default.
+
+NerdGraph can be used as an alternative to turn the setting on and off. To do so, run a mutation query on the `browserSettings` graphQL object for the `useConsentMode` property.
+
+include screen shots of both examples
+
+Once you have a privacy policy or opt-in modal integrated with your web application, then if a user clicks one of the following:
+
+1. “Yes, I consent.”
+
+    You will need to invoke use of a new API called `consent(accept?: boolean)`. When calling the [consent() API](/docs/browser/new-relic-browser/browser-apis/consent), the accept parameter is true by default.
+
+    Browser features with **autoStart: true** will send data to New Relic after consent is provided.
+
+    Browser features with **autoStart: false** will need to be invoked using the [start() API](/docs/browser/new-relic-browser/browser-apis/start) function to start deferred agent features to send data over the network.
+
+2. “No, I do not consent.”
+
+    Since the default is true, you will need to call `consent(false)`. The agent will not send data over the network for that user.
```

Comment:
```suggestion
    Since the default is true, you will need to call `consent(false)`, taking note of the false argument. The agent will not send data over the network for that user.
```